app/models/twitch_user.rb

```diff
@@ -1,41 +1,44 @@
-require 'active_support/concern'
 require 'twitch'
 
-module TwitchUser
-  extend ActiveSupport::Concern
+class TwitchUser < ApplicationRecord
+  belongs_to :user
 
-  def sync_twitch_follows!
+  def avatar
+    URI.parse(read_attribute(:avatar)).tap do |uri|
+      uri.scheme = 'https'
+    end.to_s
+  end
+
+  def sync!
+    body = JSON.parse(Twitch::User.get(twitch_id))
+
+    update(
+      twitch_id:    body['_id'],
+      name:         body['name'].downcase,
+      display_name: body['display_name'],
+      avatar:       URI.parse(body['logo'] || default_avatar).tap { |uri| uri.scheme = 'https' }.to_s
```

Comment:
I believe calling `default_avatar` like this will cause an exception to be thrown as it's now defined as a class method instead of an instance method.  I tried calling it through a debug terminal above the update call and it threw no method exception.